src/components/Menus/ContextMenu/ContextMenu.js

```diff
@@ -6,11 +6,11 @@ import './ContextMenu.scss'
 
 const ContextMenu = (props) => {
 
-  const { align, direction, menuButtonClassName, MenuButtonIcon, menuOptions, offsetX, offsetY} = props
+  const { align, direction, label, menuButtonClassName, MenuButtonIcon, menuOptions, offsetX, offsetY,} = props
```

Comment:
Could we call the `label` something like `menuButtonLabel` for clarity?

Reply:
Done in [52a3204](https://github.com/RaspberryPiFoundation/editor-ui/pull/373/commits/52a3204f4acf346cd0b5ef790a6920c97d1b53aa)